fastai/torch_imports.py

```diff
@@ -25,6 +26,15 @@ def children(m): return m if isinstance(m, (list, tuple)) else list(m.children()
 def save_model(m, p): torch.save(m.state_dict(), p)
 def load_model(m, p): m.load_state_dict(torch.load(p, map_location=lambda storage, loc: storage))
 
+def load_pre_inception(pre, f, fn):
+    m = f()
+    path = os.path.dirname(__file__)
+    if not os.path.exists(f'{path}/weights/'):
+        os.makedirs(f'{path}/weights/')
+    if pre: get_data(m.url(), f'{path}/weights/{fn}.pth')
```

Comment:
@jph00 
1) all the models that use the load_pre loader (e.g. inception4, inceptionresnet_2, resnext50, resnext101, resnext101_64, wrn) will fail without pre-creating the weights directory, we can add that logic to load_pre to unify things if you would like.

2)if you like this method of passing the url can add that logic to the existing classes.  inceptionresnet_2 has a similar model structure inception4 but the other models (eg. resnext variants + wrn) will need to be customized slightly.  the torchvision models use the torch loader directly.

Reply:
That sounds reasonable to me. Thanks for the suggestion.